src/types/database.ts

```diff
@@ -30,22 +30,28 @@ export type Database = {
     Tables: {
       issue_comments: {
         Row: {
-          commentbody: string;
+          author_id: string;
+          created_at: string;
           embedding: string;
-          id: number;
-          issuebody: string | null;
+          id: string;
+          modified_at: string;
+          plaintext: string;
         };
         Insert: {
-          commentbody: string;
+          author_id: string;
+          created_at?: string;
```

Comment:
Sync with the GitHub metadata of the comment or issue

Reply:
This is according to the schema mentioned in the issue spec, could you please be more specific with what needs to be synced ?